geomesa-filter/src/main/scala/org/locationtech/geomesa/filter/FilterHelper.scala

```diff
@@ -318,5 +320,27 @@ object FilterHelper {
       case f: Filter => Seq(f)
     }
   }
+
+  def tryMergeGeometryFilters(filts: Seq[Filter]): Seq[Filter] = {
```

Comment:
[picky] I would avoid the term "Merge" in this function name, because "merge" suggests "union" more than "intersection" (which is consistent with perusing synonyms).  Perhaps ```tryToReduceGeometryFilters```?